gdk4/src/texture.rs

```diff
@@ -14,6 +15,18 @@ pub trait TextureExtManual: 'static {
 impl<O: IsA<Texture>> TextureExtManual for O {
     fn download(&self, data: &mut [u8], stride: usize) {
         unsafe {
+            assert!(
+                stride >= 4,
+                "Stride for a CAIRO_FORMAT_ARGB32 should be >= 4"
```

Comment:
stride could be 0 if width is 0 or not?

Reply:
gdk has checks for ensuring the width & height are >0, I suppose this shouldn't happen